pkg/loadbalancerservice/controller.go

```diff
@@ -209,3 +238,59 @@ func (c *LoadbalancerServiceController) patchStatus(svc *corev1.Service, newStat
 
 	return err
 }
+
+func (c *LoadbalancerServiceController) getRouterIPAddressList() ([]string, error) {
+	configuredAddresses, err := config.GetConfiguredAddresses()
```

Comment:
In what sense are these addresses configured? Are they part of the config data the user gave us, or are they present on the host?

Reply:
These are the already present IPs in the host. Renamed both the function and the variable. I was thinking about a context where configuredXYZ means its in the host and anything coming from the user is in cfg. This can be misleading though.